include/freerdp/freerdp.h

```diff
@@ -492,6 +492,7 @@ FREERDP_API void setChannelError(rdpContext* context, UINT errorNum,
 FREERDP_API BOOL checkChannelErrorEvent(rdpContext* context);
 
 FREERDP_API const char* freerdp_nego_get_routing_token(rdpContext* context, DWORD* length);
+FREERDP_API void freerdp_update_settings(rdpContext* context);
```

Comment:
after looking through all that stuff I´d prefer a function `BOOL freerdp_settings_copy(rdpSettings* dst, const rdpSettings* src)` instead of this one. (don´t like the pointer magic, got a feeling that this is too error prone)